### PR TITLE
Machineguns Machined for Manual Use

### PR DESCRIPTION
**Pull request by EkarusRyndren**
#### Summary
```SUMMARY: Content "Adds Unmounted Heavy Weapons"```

#### Purpose of change
As it stands there are a few vehicle mounted weapons that with some skilled work could be converted for dismounted use without a bipod or tripod but aren't simply because a _normal human_ couldn't heft the thing around and/or use it effectively and/or safely.

But we aren't exactly dealing with _normal humans_ are we?

#### Describe the solution
I added these as mutant intended (But a determined player could still give them a go as a normal and/or unaided human if they're a bit crazy/desperate.)

#### Describe alternatives you've considered
I briefly considered similar modifications for some other weapons but after some discussion in the discord came to the conclusion that while similar modifications could be done it'd be far less effective. (Such as on the Minigun)

#### Additional context
It may be prudent to add some kind of self-harm tag for weapons like this if used by under-strength characters but I lack that knowledge.

### CODE REVIEW

**Comment by Rail-Runner** [comment]
Could you make use of copy-from for those, from the original (before modification) guns?

**Comment by ymber** [comment]
Adding mod locations like that isn't reasonable for that craft. Adding an improvised bullpup conversion isn't going to let you stick a bayonet on your Mk19. Also this should be implemented as a gunmod with a chance to break when you fire it and there's nothing in here making it infeasible for normal people to shoulder fire these.

**Comment by tenmillimaster** [comment]
Adding some impartial calculations here on recoil:

| Firearm                 | Recoil impulse, pound-sec | Recoil velocity, Feet/s | Recoil Energy, ft.lbf |
|-------------------------|---------------------------|-------------------------|-----------------------|
| 700 nitro Searcy Double | 17.01                     | 30.41                   | 258.73                |
| m107A1                  | 14.08                     | 14.66                   | 103.23                |
| Mk 19                   | 11.11                     | 4.41                    | 24.52                 |
| M2HB                    | 13.59                     | 5.03                    | 34.17                 |

**Comment by LaVeyanFiend** [comment]
_Fortunate Son starts playing_

**Comment by EkarusRyndren** [comment]
> Adding mod locations like that isn't reasonable for that craft. Adding an improvised bullpup conversion isn't going to let you stick a bayonet on your Mk19. Also this should be implemented as a gunmod with a chance to break when you fire it and there's nothing in here making it infeasible for normal people to shoulder fire these.

I don't see the problem with it, you're doing these conversions yourself and frankly I don't see why you wouldn't make it compatible with these mods if you're making extensive changes to an existing weapon over the course of at least six hours. Disallowing the bayonet is probably a decent idea though, the slots added though were things that were specifically mentioned as being modified.
As for doing this as a gun mod, you're making extensive changes to multiple slots, not inserting a factory-made mod into the mechanism slot.
Lastly, the sheer weight of these things loaded is gonna stop a fair number of weaker characters from being able to use them easily. Although I did notice I forgot to put the strength check in, I'll fix that in a moment.

**Comment by tenmillimaster** [comment]
> > Adding mod locations like that isn't reasonable for that craft. Adding an improvised bullpup conversion isn't going to let you stick a bayonet on your Mk19. Also this should be implemented as a gunmod with a chance to break when you fire it and there's nothing in here making it infeasible for normal people to shoulder fire these.
> 
> I don't see the problem with it, you're doing these conversions yourself and frankly I don't see why you wouldn't make it compatible with these mods if you're making extensive changes to an existing weapon over the course of at least six hours. Disallowing the bayonet is probably a decent idea though, the slots added though were things that were specifically mentioned as being modified.
> As for doing this as a gun mod, you're making extensive changes to multiple slots, not inserting a factory-made mod into the mechanism slot.
> Lastly, the sheer weight of these things loaded is gonna stop a fair number of weaker characters from being able to use them easily. Although I did notice I forgot to put the strength check in, I'll fix that in a moment.


The six hours or whatever it'd take to install a linkage and stock are get them working are *just* to do that. Adding slots, sights, bayonet lugs, whatever, that's gotta be on *top* of the modifications. There are already mods to add mod slots (e.g. sight rails, bayonet lugs, etc), so you can still make those modifications, but it'll take extra work.

This really should be a gunmod, as extensive changes are *not* being made to the base weapon. You're pulling the trigger with stick, attaching a pistol grip, and bracing the grip part against your shoulder with an attached buttpad.

To use the sights again, you'll need to install a sight rail, to install a bayonet, you have to clamp on a bayonet lug. To install a laser, you need to set up a clamp on rail, so on and so forth.

File: data/json/items/gun/50.json [hunk 1]
```diff
@@ -60,6 +60,34 @@
     "magazines": [ [ "50", [ "belt50" ] ] ],
     "flags": [ "MOUNTED_GUN" ]
   },
+  {
+    "id": "m2browningunmounted",
```
Comment (kevingranade) [inline data/json/items/gun/50.json]: This is significantly lighter than the grenade launcher, but I'm still skeptical that the 60lb gun with a 60" length is going to be remotely manageable.

**Comment by Rail-Runner** [comment]
> I don't care that it's meant for just strong mutants, normies will try to use it too. And I don't think we can adequately reflect how bad it'd be for them.

Isn't solid minimum strength value enough? Normal people simply wouldn't be able to do it.

**Pull request closed without merging**